rental.rb

```diff
@@ -1,6 +1,5 @@
 class Rental
-  attr_accessor :date
-  attr_reader :person, :book
+  attr_accessor :date, :person, :book
 
   def initialize(date, person, book)
     @date = date

```

Comment:
- [ ] Here you have to change the position of person and book since you are creating a Rental object using this parameter. 

In the `book.rb`
```ruby
Rental.new(date, self, person)
```
and in the `person.rb`
```ruby
Rental.new(date, book, self)
```

It means here you have to keep parameter like 

```ruby
def initialize(date, book, person)
```

Please fix it.

Reply:
Changed the position of person and book to
```ruby
def initialize(date, book, person)
```